ES/ESdd009.xml

```diff
@@ -30,133 +29,235 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-
-                     <incipit>
-                        <foreign xml:lang="en"/>
-                     </incipit>
-                     <explicit>
-                        <foreign xml:lang="en"/>
-                     </explicit>
-                     <note/>
+                     <locus from="3ra" to="68rb"/>
+                     <title ref="LIT1291Dersan"/>
+                     <msItem xml:id="ms_i1.1">
+                        <locus from="3ra" to="3va"/>
+                        <title type="complete" ref="LIT1291Dersan#Introduction"/>
+                     </msItem>
+                     <msItem xml:id="ms_i1.2">
+                        <locus from="3va" to="14ra"/>
+                        <title type="complete" ref="LIT1291Dersan#Monday"/>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3">
+                        <locus from="14ra" to="23rb"/>
+                        <title type="complete" ref="LIT1291Dersan#Tuesday"/>
+                     </msItem>
+                     <msItem xml:id="ms_i1.4">
+                        <locus from="23rb" to="31rb"/>
+                        <title type="complete" ref="LIT1291Dersan#Wednesday"/>
+                     </msItem>
+                     <msItem xml:id="ms_i1.5">
+                        <locus from="31rb" to="42va"/>
+                        <title type="complete" ref="LIT1291Dersan#Thursday"/>
+                     </msItem>
+                     <msItem xml:id="ms_i1.6">
+                        <locus from="42va" to="53vb"/>
+                        <title type="complete" ref="LIT1291Dersan#Friday"/>
+                     </msItem>
+                     <msItem xml:id="ms_i1.7">
+                        <locus from="53vb" to="62rb"/>
+                        <title type="complete" ref="LIT1291Dersan#Saturday"/>
+                     </msItem>
+                     <msItem xml:id="ms_i1.8">
+                        <locus from="62rb" to="68rb"/>
+                        <title type="complete" ref="LIT1291Dersan#Sunday"/>
+                     </msItem>
                   </msItem>
-                  <!--The data in the items below refers to other items. please remove this items and move the information to the appropriate place.-->
+
                   <msItem xml:id="ms_i2">
-                     <note>- The division of Text II, mainly into daily readings, differs from the
-                        more common division into monthly readings, as attested in Mss.<ref type="mss" corresp="ESgms011">GMS-011</ref>and EMML 1779.</note>
+                     <locus from="69ra" to="113ra"/>
+                     <title ref="LIT1936Mashaf"/>
+                     <note>The division of this text, mainly into daily readings, differs from the
+                        more common division into monthly readings, as attested in manuscripts <ref
+                           type="mss" corresp="ESgms011">GMS-011</ref>and <ref type="mss"
```

Comment:
space before `and`

Reply:
I think it may be good to develop the record LIT1936Mashaf (probably following ESgms011) introducing chapters there with xml:id, then when possible one could refer to these as you have beautifully done for LIT1291Dersan

Reply:
I will, but let me first do some two more mss. with this text. The division in this one seems unusual.